api/files/serializers.py

```diff
@@ -47,6 +47,14 @@ def to_internal_value(self, data):
         except StopIteration:
             self.fail('invalid_data')
 
+    def to_representation(self, value):
```

Comment:
@samchrisinger's PR (https://github.com/CenterForOpenScience/osf.io/pull/4356) fixed it in the previous implementation. I'm predicting a joyous merge. :shipit: